app/services/create_newsletter_signup.rb

```diff
@@ -0,0 +1,45 @@
+require 'sendgrid-ruby'
+include SendGrid
+
+class CreateNewsletterSignup
+  attr_accessor :newsletter_signup, :first_name, :last_name, :email
+
+  def initialize(first_name:, last_name:, email:)
+    @newsletter_signup = {
+      first_name: first_name,
+      last_name: last_name,
+      email: email
```

Comment:
suggestion: don't set this equal to a hash here. just set the attr accessor vars in the initializer, then they will be available to you without the @ later on